apps/mull-api/src/database/seeders/database.seeder.ts

```diff
@@ -0,0 +1,29 @@
+import { Connection } from 'typeorm';
+import { Factory, Seeder } from 'typeorm-seeding';
+import { Event } from '../../app/entities';
+
+/**
+ * Simple algorithm that will add the next 3 users as a friend
+ * @param connection
+ */
+const createFriends = async (connection: Connection) => {
```

Comment:
The friends table is not being populated

Reply:
fixed